.cirrus.yml

```diff
@@ -65,10 +65,13 @@ task:
     # Add a user for rootless tests
     useradd -u2000 -m -d/home/rootless -s/bin/bash rootless
     # Allow root and rootless itself to execute `ssh rootless@localhost` in tests/rootless.sh
-    ssh-keygen -t ecdsa -N "" -f /root/rootless.key
+    ls -alh /root
+    chmod +w /root
+    mkdir -p /root/.ssh
```

Comment:
1. `-p` is not needed.
2. `-m 0700` is probably needed.

Reply:
Maybe I was wrong, `-p` is needed when /root/.ssh already exists.

But anyway, PTAL #4728